### PR TITLE
Build job update

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Add koji build to supported jobs and update triggers for build jobs.
- I've marked koji builds as `in preview` to make it sure it's not fully supported.

---

- [x] I'll regenerate html the content after the review.

### CODE REVIEW

File: content/docs/configuration.md [hunk 1]
```diff
@@ -282,6 +284,28 @@ With this configuration, you'll get builds in all stable fedora releases
...
+
+Required metadata:
+
+* **branch** -- the name of the branch we want to build for when using **commit** trigger
+
+Optional metadata:
+
+* **targets** -- a (list of) targets we want to build for,
+  list of supported targets can be listed using with `koji list-targets`.
+* **scratch* -- defaults to `false`, use to create scratch (test) builds
```
Comment (csomh) [inline content/docs/configuration.md:306]:
```suggestion
* **scratch** -- defaults to `false`, use to create scratch (test) builds
```

File: content/docs/configuration.md [hunk 2]
```diff
@@ -282,6 +284,28 @@ With this configuration, you'll get builds in all stable fedora releases
...
+* **release** -- check out content of the tag associated with the release
+
+Required metadata:
+
+* **branch** -- the name of the branch we want to build for when using **commit** trigger
+
+Optional metadata:
+
+* **targets** -- a (list of) targets we want to build for,
+  list of supported targets can be listed using with `koji list-targets`.
```
Comment (csomh) [inline content/docs/configuration.md:305]:
```suggestion
  list of supported targets can be listed using with `koji list-targets`
```

**Comment by lachmanfrantisek** [comment]
> > 
> 
> Or I can push the re-generate commit here to avoid all the (different hugo version caused) whitespaces added/removed noise.

@jpopelka That would be nice, thanks!

**Comment by jpopelka** [comment]
force pushed :-)

**Comment by lachmanfrantisek** [comment]
Thanks!

**Pull request merged**